.rubocop.yml

```diff
@@ -55,6 +55,9 @@ Naming/VariableNumber:
 MultilineBlockChain:
   Enabled: false
 
+Style/MultilineMethodCallIndentation:
+  EnforcedStyle: indented
```

Comment:
I personally prefer this style, and feel it helps keep the lines shorter than the default

Reply:
I prefer `EnforcedStyle: aligned`
which aligns better with functional programming style:
👍 
```
# good, EnforcedStyle: aligned
Thing.a
     .b
     .c 
```

vs indented
👎 
```
while a
    .b
  something
end
```

Reply:
I can change it back 👍